bigquery/google/cloud/bigquery/query.py

```diff
@@ -31,7 +31,6 @@ class _SyncQueryConfiguration(object):
     Values which are ``None`` -> server defaults.
     """
     _default_dataset = None
-    _dry_run = None
```

Comment:
I'm not sure what you mean: it is right there on line 10 of your screenshot.  The `query` field in that resource is just the `SELECT` statement (a string).